WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -42,9 +42,10 @@ final class ProductsViewController: UIViewController {
 
     /// Top toolbar that shows the sort and filter CTAs.
     ///
-    private lazy var toolbar: UIView = {
-        return createToolbar()
-    }()
+    @IBOutlet weak var toolbar: ToolbarView!
```

Comment:
nit: could this be private?

Reply:
Indeed! :) Done in https://github.com/woocommerce/woocommerce-ios/pull/6698/commits/4812d6a8a98e78adc30468408cfad43bc1703083